validator_client/src/block_service.rs

```diff
@@ -584,7 +511,7 @@ impl<T: SlotClock + 'static, E: EthSpec> BlockService<T, E> {
                         &metrics::BLOCK_SERVICE_TIMES,
                         &[metrics::BEACON_BLOCK_HTTP_GET],
                     );
-                    let block_response = Self::get_validator_block_v3(
+                    let block_response = Self::get_validator_block(
```

Comment:
Not part of this PR, I think it may be unnecessary to re-map the `BlockError` here?

https://github.com/sigp/lighthouse/pull/5292/files#diff-9397634273d7e6a89631ace49f8f1cfe7f083db2b556f1dee86b76184b7618e6R524-R529